src/simtools/simtel/pulse_shapes.py

```diff
@@ -0,0 +1,154 @@
+"""Pulse shape computations for light emission simulations for flasher."""
+
+from __future__ import annotations
+
+import logging
+
+import numpy as np
+from scipy.optimize import brentq, minimize_scalar
+from scipy.signal import fftconvolve
+
+_logger = logging.getLogger(__name__)
+
+
+def _rise_width(t, y, y_low=0.1, y_high=0.9):
```

Comment:
Small note - no direct need to do something. But _rise_width and _fall_width looks almost identical.